dashboard/db/migrate/20170215183924_modify_script_level_properties.rb

```diff
@@ -0,0 +1,24 @@
+class ModifyScriptLevelProperties < ActiveRecord::Migration[5.0]
+  # ScriptLevel.properties is currently a hash representing variant levels. We'd
+  # like the property hash to be more generic, so we're moving the variants to
+  # exist under their own key within properties
+
+  def up
+    ScriptLevel.where.not(properties: nil).each do |script_level|
+      obj = JSON.parse script_level.properties
+      script_level.properties = { variants: obj }.to_json
+      script_level.save
```

Comment:
Any reason not to write this as `script_level.update!(properties: { variants: obj }.to_json)`? Also below.

Note the exclamation, consider rescuing the exception.